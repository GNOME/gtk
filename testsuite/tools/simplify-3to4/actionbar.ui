<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <object class="GtkActionBar">
    <child>
      <object class="GtkButton" id="button">
        <property name="icon-name">document-new</property>
      </object>
      <packing>
        <property name="pack-type">start</property>
      </packing>
    </child>
  </object>
</interface>
