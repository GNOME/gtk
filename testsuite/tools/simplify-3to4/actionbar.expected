<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <requires lib="gtk" version="4.0"/>
  <object class="GtkActionBar">
    <property name="visible">0</property>
    <child type="start">
      <object class="GtkButton" id="button">
        <property name="visible">0</property>
        <property name="icon-name">document-new</property>
      </object>
    </child>
  </object>
</interface>
